index.html

```diff
@@ -1179,6 +1179,24 @@ <h2>A project for first-time contributions</h2>
         </div>
         <!-- ________ Chisom's card END ________  -->
 
+        <!-- ________ ap1459's card END ________  -->
+         <div class="card">
+
+            <p class="name">AP1459</p>
+            <p class="contact"><i class="fa fa-twitter"></i> <a href="YOUR twitter account link" target="_blank">@BIS3300_1718_20</a></p>
```

Comment:
You forgot to add your twitter URL in the `href`. It should look something like this `<a href="https://twitter.com/Syknapse"` but with your user name instead of 'Syknapse'